### ISSUE TITLE
media-libs/bankstown-lv2 support for slotted Rust

### ISSUE THREAD

**Issue by qdotme** [chadmed/asahi-overlay#124]
Removing virutal/rust as masked upstream leads to dependency check failure for media-libs/bankstown-lv2

```
- virtual/rust-1.82.0::gentoo (masked by: package.mask)
/var/db/repos/gentoo/profiles/package.mask:
# Matt Jolly <kangie@gentoo.org> (2024-11-09)
# The legacy Rust ebuilds are no longer maintained and have been
# replaced by the new slotted Rust. Removal in 30 days.
```

**Comment by chadmed** [comment]
Yeah, didn't realise the rust SLOT changed landed today... Working on it now.

**Comment by chadmed** [comment]
Fixed by e3f97a6
